.gitignore

```diff
@@ -1,21 +0,0 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
this file is kind of important, where did it go?

Reply:
re-added!